gateway/conf/migration/migrations.ts

```diff
@@ -0,0 +1,17 @@
+import * as utils from '../../src/services/config-manager-v2';
+
+/*
+ * This file defines migration functions for each version.
+ * Note: type utils.Migration = (configRootFullPath: string,
+ *                         configRootTemplateFullPath: string) => void;
+ */
+
+export const updateToVersion1: utils.Migration = (
```

Comment:
Since this file is not supposed to be changed by the user, but by gateway developers - it should not be placed inside the conf folder. Lets move it to `gateway/src/services/config-migration/migrations.ts`.